src/test/resources/pdfs/PdfContentImporter/pdfContentImporterTest-pdfs.bib

```diff
@@ -0,0 +1,128 @@
+@inproceedings{Kriha2018Teaching,
+author = {Walter Kriha and Tobias Jordine}
```

Comment:
Note: This is INVALID BibTeX. All commans are missing.

In the future: Please try to open the file in JabRef.